### PR TITLE
Translation Delivery

### PR DESCRIPTION
**Pull request by service-tip-git**
Translation Delivery

### CODE REVIEW

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them.

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -2138,6 +2149,10 @@
 
 "Statistics_Info_Subtitle" = "Explanation of Statistics";
 
+"Statistics_Info_PandemicRadar_Title" = "Pandemic Radar";
+
+"Statistics_Info_PandemicRadar_Text" = "A variety of statistics on the coronavirus pandemic, which is provided on a Robert Koch Institute website.\nPlease note that any data used when opening the website will be deducted from your data plan and that the website is not designed to be barrier-free.";
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:2154]:
```suggestion
"Statistics_Info_PandemicRadar_Text" = "A variety of statistics on the coronavirus pandemic, which is provided on a Robert Koch Institute website.\nPlease note that any data used when opening the website will be deducted from your data plan, and that the website is not designed to be barrier-free.";
```

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -1656,10 +1656,10 @@
...
+"NewVersionFeature_228_pandemic_radar_description" = "The Corona-Warn-App now contains a “Pandemic Radar” tile, which takes you directly to a Robert Koch Institute website (in German) when tapped. The Pandemic Radar contains a variety of statistics on the coronavirus pandemic, which are updated every working day. Please note that any data used when opening the website will be deducted from your data plan and that the website is not designed to be barrier-free.";
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:1662]:
```suggestion
"NewVersionFeature_228_pandemic_radar_description" = "The Corona-Warn-App now contains a “Pandemic Radar” tile, which takes you directly to a Robert Koch Institute website (in German) when tapped. The Pandemic Radar contains a variety of statistics on the coronavirus pandemic, which are updated every working day. Please note that any data used when opening the website will be deducted from your data plan, and that the website is not designed to be barrier-free.";
```

**Review by flxschmidt — APPROVED** [review]
Long sentence, I would separate sub-clause with a comma. But it's not necessary.

**Review by puneetmahali1 — APPROVED** [review]

**Pull request merged**